GitUI/UserControls/FileStatusList.cs

```diff
@@ -727,28 +675,9 @@ public void SetSelectedIndex(int idx, bool notify)
 
         public int SetSelectionFilter(string selectionFilter)
         {
-            return SelectFiles(RegexForSelecting(selectionFilter));
-
-            int SelectFiles(Regex regex)
-            {
-                try
-                {
-                    SuspendLayout();
-
-                    var i = 0;
-                    foreach (var item in AllItems)
-                    {
-                        FileStatusListView.Items[i].Selected = regex.IsMatch(item.Name);
-                        i++;
-                    }
-
-                    return FileStatusListView.SelectedIndices.Count;
-                }
-                finally
-                {
-                    ResumeLayout(true);
-                }
-            }
+            var regex = RegexForSelecting(selectionFilter);
+            SelectItems(item => regex.IsMatch(item.Name));
+            return FileStatusListView.SelectedIndices.Count;
```

Comment:
Almost hidden feature: Have you ever hit `Ctrl+F` in the Unstage Files List?

Reply:
👀 
No... That feature looks very strange.
We have a filter dropdown that is much more useful. The same filter should be in the Staged list too (and the Diff form too).

I would recommend that feature for removal.